cypress/support/commands.js

```diff
@@ -1,19 +1,23 @@
 Cypress.Commands.add("login", () => {
-  //cy.visit(Cypress.env("baseUrl"));
-  const username = Cypress.env("username");
+  const email = Cypress.env("email");
   const password = Cypress.env("password");
 
-  cy.get("#email").clear().type(username);
+  cy.get("#email").clear().type(email);
   cy.get("#password").clear().type(password);
-  cy.get("[type='submit']").click();
+  cy.get("#loginButton").click();
   cy.url().should("include", "/overview");
 });
 
 Cypress.Commands.add("preserveCookies", () => {
-  Cypress.Cookies.preserveOnce("Agentcis","laravel_token");
+  Cypress.Cookies.preserveOnce("Agentcis", "laravel_token");
 });
 
 Cypress.Commands.add("visitMainPage", () => {
   cy.visit(Cypress.env("baseUrl"));
 });
 
+Cypress.Commands.add("emptyValidation", (selectors) => {
```

Comment:
please use generic command naming as it will be used in other tests as well .So ,rename accordingly